src/generator.cs

```diff
@@ -2102,7 +2104,7 @@ public void Go ()
 			marshal_types.Add (new MarshalType (TypeManager.ABRecord, create: "ABRecord.FromHandle("));
 		}
 		if (Frameworks.HaveCoreVideo)
-			marshal_types.Add (TypeManager.CVPixelBuffer);
+			marshal_types.Add (new MarshalType (TypeManager.CVPixelBuffer, create: "Runtime.GetINativeObject<CVPixelBuffer> (", closingCreate: ", false)")); // owns `false` like ptr ctor https://git.io/vHvLj
```

Comment:
The gitio.io link points to a specific location in `master`, which is a moving target (changing code in that file will cause this link to point somewhere else, which makes the comment inexplicable).

You can manually change `master` in the link to a specific hash (say 6f68ab6f79c5f1d96d2cbb1e697330623164e46d):

https://github.com/xamarin/xamarin-macios/blob/6f68ab6f79c5f1d96d2cbb1e697330623164e46d/src/CoreVideo/CVBuffer.cs#L74-L90

and your link target won't change in the future.

Also personally I don't like short links, because you have to click on them to see what they're pointing too (they can point anywhere, including somewhere with malware; and they're useless while working offline).